native/src/main/java/io/ballerina/stdlib/etl/utils/ErrorUtils.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ * 
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.etl.utils;
+
+import io.ballerina.runtime.api.creators.ErrorCreator;
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BError;
+import io.ballerina.runtime.api.values.BString;
+
+/**
+ * Represents the errors of ETL module.
+ */
+public class ErrorUtils {
+
+    private ErrorUtils() {
+    }
+
+    public static BError createCommonFieldNotFoundError(int datasetIndex) {
+        return ErrorCreator.createError(
+                StringUtils.fromString(String.format("The dataset%d does not contain the field - '%s'", datasetIndex,
```

Comment:
Shouldn't we have a space here?
```suggestion
                StringUtils.fromString(String.format("The dataset %d does not contain the field - '%s'", datasetIndex,
```